dashboard/BUILD

```diff
@@ -20,4 +19,24 @@ py_binary(
         "//marketplace",
     ],
     visibility = ["//visibility:public"],
+)
+
+py_binary(
```

Comment:
When it is a library and not something that you want to run, you should create a py_library instead of a py_binary. It is the same for charts and figures.

Reply:
My fault :(